tdrs-backend/tdpservice/settings/cloudgov.py

```diff
@@ -24,6 +24,28 @@ def get_cloudgov_service_creds_by_instance_name(services, instance_name):
     )
 
 
+def get_cloudgov_broker_db_numbers(cloudgov_name):
+    """
+    Get the appropriate redis broker db numbers for an environment.
+
+    Returns a tuple of (broker_db_number, results_db_number)
+    """
+    match cloudgov_name:
+        case 'raft':
+            return ('0', '1')
+        case 'qasp':
+            return ('2', '3')
+        case 'a11y':
+            return ('4', '5')
+        case 'develop':
+            return ('0', '1')
+        case 'staging':
+            return ('2', '3')
+        case 'prod':
+            return ('0', '1')
+    return ('0', '1')
```

Comment:
@elipe17 by default that's `0` for the broker, `1` for the results - here's the function that defines which they are for each app env in the respective spaces. that should prevent cross-contamination as you say. i tested this by deploying to both a11y and raft and putting a few unique files through each

Reply:
right, the database/queue number (`/1`) is a feature of redis. redis docs page is giving me an error right now, but here are some docs for a redis client that show it's availability - https://github.com/redis/lettuce/wiki/Redis-URI-and-connection-details#uri-syntax
i think celery calls it `virtual_host` here - https://docs.celeryq.dev/en/latest/userguide/configuration.html#std-setting-broker_url

as far as i can tell, redis just creates the database in memory, on the fly. as long as celery and django for that env use the same database, they should communicate with each other properly.